jbpm-db-scripts/src/main/resources/db/ddl-scripts/db2/db2-jbpm-schema.sql

```diff
@@ -1,818 +1,191 @@
-    create index IDX_BAMTaskSumm_status on BAMTaskSummary(status);
-    create index IDX_BAMTaskSumm_taskId on BAMTaskSummary(taskId);
-    create index IDX_BAMTaskSumm_taskName on BAMTaskSummary(taskName);
-    create index IDX_BAMTaskSumm_userId on BAMTaskSummary(userId);
-    
-    create index IDX_PInstLog_duration on ProcessInstanceLog(duration);
-    create index IDX_PInstLog_end_date on ProcessInstanceLog(end_date);
-    create index IDX_PInstLog_extId on ProcessInstanceLog(externalId);
-    create index IDX_PInstLog_user_identity on ProcessInstanceLog(user_identity);
-    create index IDX_PInstLog_outcome on ProcessInstanceLog(outcome);
-    create index IDX_PInstLog_parentPInstId on ProcessInstanceLog(parentProcessInstanceId);
-    create index IDX_PInstLog_pId on ProcessInstanceLog(processId);
-    create index IDX_PInstLog_pInsteDescr on ProcessInstanceLog(processInstanceDescription);
-    create index IDX_PInstLog_pInstId on ProcessInstanceLog(processInstanceId);
-    create index IDX_PInstLog_pName on ProcessInstanceLog(processName);
-    create index IDX_PInstLog_pVersion on ProcessInstanceLog(processVersion);
-    create index IDX_PInstLog_start_date on ProcessInstanceLog(start_date);
-    create index IDX_PInstLog_status on ProcessInstanceLog(status);
-    create index IDX_PInstLog_correlation on ProcessInstanceLog(correlationKey);
-
-    create index IDX_VInstLog_pInstId on VariableInstanceLog(processInstanceId);
-    create index IDX_VInstLog_varId on VariableInstanceLog(variableId);
-    create index IDX_VInstLog_pId on VariableInstanceLog(processId);
-
-    create index IDX_NInstLog_pInstId on NodeInstanceLog(processInstanceId);
-    create index IDX_NInstLog_nodeType on NodeInstanceLog(nodeType);
-    create index IDX_NInstLog_pId on NodeInstanceLog(processId);
-
-    create index IDX_ErrorInfo_pInstId on ExecutionErrorInfo(PROCESS_INST_ID);
-    create index IDX_ErrorInfo_errorAck on ExecutionErrorInfo(ERROR_ACK);
-
-    create index IDX_AuditTaskImpl_taskId on AuditTaskImpl(taskId);
-    create index IDX_AuditTaskImpl_pInstId on AuditTaskImpl(processInstanceId);
-    create index IDX_AuditTaskImpl_workItemId on AuditTaskImpl(workItemId);
-    create index IDX_AuditTaskImpl_name on AuditTaskImpl(name);
-    create index IDX_AuditTaskImpl_processId on AuditTaskImpl(processId);
-    create index IDX_AuditTaskImpl_status on AuditTaskImpl(status);
-
-    create index IDX_TaskVariableImpl_taskId on TaskVariableImpl(taskId);
-    create index IDX_TaskVariableImpl_pInstId on TaskVariableImpl(processInstanceId);
-    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
\ No newline at end of file
+create table Attachment (id bigint generated by default as identity, accessType integer, attachedAt timestamp, attachmentContentId bigint not null, contentType varchar(255), name varchar(255), attachment_size integer, attachedBy_id varchar(255), TaskData_Attachments_Id bigint, primary key (id));
```

Comment:
I think it should be good to keep the format as we already have as it makes easier to read - what do you think?